lib/Doctrine/ORM/Query/Filter/SQLFilter.php

```diff
@@ -24,7 +24,7 @@ abstract class SQLFilter
      *
      * @var EntityManagerInterface
      */
-    private $em;
+    protected $em;
```

Comment:
There is no reason for the entity manager to be writable. I would be more ok with a `final protected` accessor.

Reply:
:thinking: it's an object, so I guess it would be writable anyway, unless we clone it in the accessor, which is probably not desirable.

Reply:
But I guess at least you will not be able to replace `$em` with another instance, and since that object is already shared with a lot of things, and you access it in your filter, it's not a big deal to give access to it. Please use a `final protected` accessor.